src/charts/histogramBoxPlot/onResize.js

```diff
@@ -39,6 +39,60 @@ export default function onResize() {
         const quantile = quantiles[item];
         quantile.quantile = d3quantile(this.values, quantile.probability);
 
+		// PLOT OUTLIERS
```

Comment:
Probably best to pull the point rendering out of the `for(item in quantiles)` loop and render a bit more directly. 

Consider something like this to find the outliers:

```
var outliers = this.values.filter(function(f){
 var low_outlier = f < quantiles.filter(q=>q.probability==0.05)[0]["quantile"] )
 var high_outlier = f >quantiles.filter(q=>q.probability==0.95)[0]["quantile"] )
 return low_outlier || high_outlier
)
```

And then render them using the standard d3 workflow (select/data/enter/append), that reuses much of your original code. 

```
var chart=this;
this.svg
.selectAll("line.outlier")
.data(outliers)
.enter()
.append("line")
.attr(function(d){return {
   class: "statistic outlier",
     x1: chart.x(d),
     y1: chart.plot_height * 1.07,
     x2: chart.x(d),
     y2: (chart.plot_height + chart.config.boxPlotHeight) / 1.07
   })
   .style(...)

```